api/tests/routes/backoffice/pro_users_test.py

```diff
@@ -320,6 +320,20 @@ def test_no_user_offerers(self, authenticated_client, pro_user):
             response = authenticated_client.get(url)
         assert response.status_code == 200
 
+    def test_get_user_offerers_in_different_offerer_status(self, authenticated_client, pro_user):
+        offerers_factories.UserOffererFactory(user=pro_user, offerer=offerers_factories.PendingOffererFactory())
+        url = url_for(self.endpoint, user_id=pro_user.id)
+
+        with assert_no_duplicated_queries():
+            response = authenticated_client.get(url)
+
+        assert response.status_code == 200
```

Comment:
Depuis le point de départ de ta branche, j'ai mergé une PR utilisant `assert_num_queries` partout afin de détecter les éventuelles requêtes en trop, peux-tu _rebase_ et remplacer ?
```suggestion
        with assert_num_queries(self.expected_num_queries)::
            response = authenticated_client.get(url)
            assert response.status_code == 200

```